query/identity.sp

```diff
@@ -71,6 +71,44 @@ query "identity_authentication_password_policy_strong_min_length_14" {
   EOQ
 }
 
+query "identity_allow_group_administrators_manage_all_resources_in_tenancy" {
+  sql = <<-EOQ
+    select
```

Comment:
This query should show result on the tenant level, If any group except administrators has got `manage all resources` permission then it should go in alarm .